driver/bpf/fillers.h

```diff
@@ -4745,4 +4788,52 @@ FILLER(netif_receive_skb_e, false)
 	return 0;
 }
 
+FILLER(tcp_send_reset_e, false)
+{
+	struct tcp_reset_args *args;
+	struct sock *sk;
+	long size = 0;
+	args = (struct tcp_reset_args*) data->ctx;
```

Comment:
these lines can be optimized into one line
`struct tcp_reset_args *args  = (struct tcp_reset_args*) data->ctx;`
FILLER(tcp_receive_reset_e, false) is similar